web/src/rest.ts

```diff
@@ -161,8 +162,7 @@ const dandiRest = new Vue({
       }
     },
     async mostRecentVersion(identifier: string) {
-      // TODO: find a way to do this with fewer requests
-      const count = (await this.versions(identifier))?.count;
+      const count = store.state.dandiset.versionCount; // todo - this.$store.state is undefined?
```

Comment:
Unable to resolve cyclic dependency between store.js and rest.js. Alternate approach tried:
```
computed: {
    versionCount(): Number {
      return this.$store?.direct.state.dandiset.versionCount;
    },
}
```

```
async mostRecentVersion(identifier: string) {
      const count = this.versionCount;
```

Reply:
I think we should just avoid using the store here in the first place. We can just pass `versionCount` as a parameter to this function.

Reply:
What is the benefit of passing a parameter over using a store? In other words, does using a store create an issue?

Reply:
I don't think we should be using the store in this file at all. Currently there's a separation of the store and the API wrapper, and I think to reduce complexity we should keep it that way. Everything that's currently being done (tracking `versionCount` in the store) can still be done, but we can simply just pass that value as an argument to `mostRecentVersion`, instead of importing the store and then reading it.

If this PR is updated to add sorting by most recent, then I suspect the use of `mostRecentVersion` will become unnecessary anyway, since we can just grab it from the response and save it directly in the store.

Reply:
Got it!